### PR TITLE
added design kit info

### PR DESCRIPTION
**Pull request by stacymcauliffe**
Hi guys! I updated the get started section to include additional info about the Sketch design kit. I tried to author in the same style Christie was using throughout. 

@gdoyle1 - hoping you can review to make sure the steps are correct. 

Thanks

### CODE REVIEW

File: packages/patternfly-4/content/get-started/gettingstarted-design.md [hunk 1]
```diff
@@ -6,7 +6,29 @@ path: "/get-started/designers"
...
 <p>Before you begin, check out <a href="/get-started/about">About PatternFly 4</a> to get acquainted with the basic elements of the design system.</p>
 
-<h1 class="pf-c-title pf-m-3xl pf-u-mt-xl">Designing with PatternFly 4</h1>
+<h1 class="pf-c-title pf-m-3xl pf-u-mt-xl">Get set up</h1>
+
+<h1 class="pf-c-title pf-m-2xl pf-u-mt-xl">Download and install the PatterFly 4 design kit</h1>
+
+<p>The design kit is a collection of assets that make it easy for you to create high-fidelity design mockups using PatternFly 4 components.</p>
+
+<p>The design kit works with Sketch libraries and templates. To use it, you'll need to have <div class="link"><a href="https://www.sketch.com/">Sketch</a> version 47 or above.</p>
```
Comment (gdoyle1) [inline packages/patternfly-4/content/get-started/gettingstarted-design.md]: "The design kit includes Sketch libraries and templates" I think makes more sense here

File: packages/patternfly-4/content/get-started/gettingstarted-design.md [hunk 2]
```diff
@@ -6,7 +6,29 @@ path: "/get-started/designers"
...
+
+<h1 class="pf-c-title pf-m-xl pf-u-mt-xl">Download the design kit files:</h1>
+<p><b>Add the Sketch library</b></p>
+<ol>
+<li>Access the <div class="link"><a href="https://sketch.cloud/s/wxoZ9">PatternFly 4 design library</a> on Sketch cloud.</li>
+<li>Click <b>Add library to Sketch</b> on the bottom right to add the library. You'll be prompted with a modal. Click <b>Add library to Sketch</b> again. Sketch will show you the library in your preferences window. You're all set!</li>
+
```
Comment (gdoyle1) [inline packages/patternfly-4/content/get-started/gettingstarted-design.md]: Can we add something at the end saying it'll live in the "symbols" button in the sketch toolbar? And that's how you access it

File: packages/patternfly-4/content/get-started/gettingstarted-design.md (hunk 1 above)
Comment (dgutride) [inline packages/patternfly-4/content/get-started/gettingstarted-design.md]: Missing a closing </div> here

File: packages/patternfly-4/content/get-started/gettingstarted-design.md [hunk 3]
```diff
@@ -6,7 +6,29 @@ path: "/get-started/designers"
...
+
+<p><b>Add the Sketch template</b></p>
+<ol>
```
Comment (dgutride) [inline packages/patternfly-4/content/get-started/gettingstarted-design.md]: you have to close the previous <ol> before you open another one unless you intended this to be nested (then it should be inside an <li>

File: packages/patternfly-4/content/get-started/gettingstarted-design.md [hunk 4]
```diff
@@ -6,7 +6,29 @@ path: "/get-started/designers"
...
+<li>Access the <div class="link"><a href="https://sketch.cloud/s/wxydj">PatternFly 4 design template</a> on Sketch cloud.</li>
+<li>Click <b>Download Document</b> on the bottom right to add the library.</li>
+<li>Navigate to the file you just downloaded, <code>PatternFly 4 Template.sketch</code>, and open it.</li>
+<li>Navigate to <code>File > Save as Template...</code> and name your file whatever makes sense to you. We recommend something simple like <code>PatternFly 4</code>." You can now access the PatternFly 4 template from <code>File > New from Template...</code>.</li>  
+
```
Comment (dgutride) [inline packages/patternfly-4/content/get-started/gettingstarted-design.md]: did the list ever close?  need a final </ol> here

**Comment by stacymcauliffe** [comment]
Added this content in a different format and PR

**Pull request closed without merging**